jujugui/static/gui/src/app/assets/js/dev.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+window.juju_config = require('./config');
+window.YUI = require('../../../../build/app/assets/javascripts/yui/yui/yui').YUI;
+
+// Now that all of the above JS is loaded we can define the real start
+// function which will be picked up by the setTimeout, and the app will
+// start.
+window.startTheApp = function() {
+  YUI().use([], function(Y) {
+    window.yui = Y;
+    const JujuGUI = require('../../init');
```

Comment:
This require was in the index.html template, but Parcel can't handle requires inside html templates so I moved this code to an external file.